### ISSUE TITLE
Customizable delay and reset to default state for window covering

### ISSUE THREAD

**Issue by sveeen** [justjam2013/homebridge-virtual-accessories#42]
Two little wishes from me: 
1. Can you add a free delay input field? What is the minimum? 1 sec or even less (< 1 sec)?
2. Would it be possible to reset the state to “closed” or to the default state after X seconds (I choose "closed" because of the Siri command "Open trash"

**Comment by justjam2013** [comment]
For clarification:
> add a free delay input field

Is this an open/close delay? I.e the time it takes to go from `Open` to `Closed` (and vice versa) during which it shows as `Opening` or `Closing`?

**Comment by sveeen** [comment]
Yes, that would be great. I use it for my trash and at the moment it opens with a delay of 3 seconds or more.

1 second or less would Great - better a input

**Comment by justjam2013** [comment]
> "I use [your plugin] for my trash"

Just what every plugin author wants to hear! 🤣🤣🤣

**Comment by justjam2013** [comment]
I have considered the reset request. Blinds and shades normally open and close either based on the outside light or based on sleep requirements. So I don't think it makes sense for a window covering to reset.

I realize that this isn't your use case, but in a manner of speaking you are using this accessory in an unconventional way. Can I suggest creating a switch with a reset timer? Then in an automation when the trash bin is open, it flips the switch, starting the timer. When the time runs out, the switch closes and a second automation closes the trash bin.
Let me know what you think.

I have added the transition delay field, which will require a manual fix of the configuration in the JSON editor and I will publish this shortly.

**Comment by justjam2013** [comment]
**Update:** published v1.2.2 with configurable open/close transition delay.

**Comment by justjam2013** [comment]
Completed.
